lib/guest/linux.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 const (
-	linuxGuestID = "other3xLinux64Guest"
+	linuxAlternateGuestName = "vSphere Integrated Containers - Container 0.9.0"
```

Comment:
we have a version definition, which is dynamically bound with release, build, commit, etc. https://github.com/vmware/vic/blob/master/pkg/version/version.go